java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -996,7 +997,8 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_stringConcatenation(
 
 JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_binaryOpVV(JNIEnv *env, jclass,
                                                                     jlong lhs_view, jlong rhs_view,
-                                                                    jint int_op, jint out_dtype) {
+                                                                    jint int_op, jint out_dtype,
+                                                                    jint scale) {
```

Comment:
We need to use the scale not just pass it in.

Reply:
done